service/topology_coordinator.cc

```diff
@@ -2898,6 +2898,12 @@ future<> topology_coordinator::run() {
                 co_await await_event();
                 rtlogger.debug("topology coordinator fiber got an event");
             }
+            co_await utils::get_local_injector().inject("wait-before-handling-keyspace-rf-change", [] (auto& handler) -> future<> {
```

Comment:
Let's name this injection more generically. It has nothing to do with keyspace RF change -- it's a general injection to sleep after getting event in topology coordinator fiber. Currently it's only used in test for keyspace rf change, but we may want to reuse it in other tests in other contexts in the future.

Reply:
Right, changed to `wait-after-topology-coordinator-gets-event`.